test/app/db/migrate/20200723100326_init.rb

```diff
@@ -3,7 +3,10 @@ def change
     create_table :users do |t|
       t.string :name, null: false
       t.integer :age, null: false
-      t.string :status, null: false
+      t.integer :status, null: false
+      t.string :phone_numbers
+      t.string :contact_info
+      t.string :family_tree
```

Comment:
I found https://github.com/pocke/rbs_rails/pull/293 did not add columns to the migration file.